### ISSUE TITLE
Tabbing pointer in cx-tab is not insufficient visible for a11y

### ISSUE THREAD

**Issue by Pucek9** [SAP/spartacus#8334]
Steps to reproduce:
1 Open eg. http://localhost:4200/powertools-spa/en/USD/organization/cost-centers
2 Navigate by tab key for focuse any link

Observed result: Name link has visible layer only on the right side, for unit link it's not visible at all.

Also out of table - I can't see that effect for link icons like '+', '>', '<', 'x' (partially), 'open in new window icon'.

Please dont care about tab-focusing stuff from invisible views, it should be done in #8208
